gatewayconfig/gatewayconfig_app.py

```diff
@@ -91,9 +91,14 @@ def init_nmcli(self):
         nmcli_custom.disable_use_sudo()
 
     def init_gpio(self):
-        self.user_button = Button(self.get_button_pin(), hold_time=USER_BUTTON_HOLD_SECONDS)
-        self.user_button.when_held= self.start_bluetooth_advertisement
-        self.status_led = LED(self.get_status_led_pin())
+        if is_raspberry_pi():
+            self.user_button = Button(self.get_button_pin(), hold_time=USER_BUTTON_HOLD_SECONDS)
+            self.user_button.when_held= self.start_bluetooth_advertisement
+            self.status_led = LED(self.get_status_led_pin())
+        else:
+            logger.warn("LEDs disabled. GPIO not yet supported on this device.")
```

Comment:
LEDs and buttons

Reply:
Fixed: https://github.com/NebraLtd/hm-config/pull/141/commits/b0d88035fb7eb7344637119439d46539189a4868#diff-8f3c7cad01ea4ab5283fb98a71afe99f7f4a170a2387b7477d6755990a11f577R99